### ISSUE TITLE
[BUG]: Baklava select input popover close event trigger problem

### ISSUE THREAD

**Issue by oemolla** [Trendyol/baklava#802]
### Issue description

When typing in the bl select input, pressing the space bar causes the popover to close.

### Media & Screenshots

![last](https://github.com/Trendyol/baklava/assets/12650525/2e9f6490-6004-44bd-8bca-41409779968d)

### Priority this issue should have

Low (slightly annoying)

### Please review the checkboxes that are applicable.

- [ ] This issue is directly related to React

**Comment by erbilnas** [comment]
merged https://github.com/Trendyol/baklava/pull/796
